requirements.txt

```diff
@@ -47,10 +45,8 @@ pyparsing==2.4.7
 pysolr==3.10.0b1
 pysndfile==1.4.3
 python-cjson==1.1.0
-python-memcached==1.53
 pytz==2020.1
 PyYAML==3.11
-raven==6.10.0
```

Comment:
I think we need this for sentry reports right? @alastair

Reply:
Raven is deprecated in favor of Sentry-Python (`sentry-sdk`)

Reply:
ah right, I see, I forgot that we made that change :)